docs/DNS.md

```diff
@@ -28,3 +28,14 @@ uds run --set COREDNS_OVERRIDES="$COREDNS_OVERRIDES"
 ## Nginx Configuration
 
 Additionally, the package includes Nginx configuration that assumes the use of `uds.dev` as the base domain. This configuration is tailored to support the development environment setup, ensuring that Nginx correctly handles requests and routes them within the cluster, based on the `uds.dev` domain.
+
+### Nginx Additional Gateway Support
+
+You can add an additional Gateway to the Nginx configuration by overriding the helm value `customGateway.domainName` with the desired domain name.
+
+i.e. to add a custom gateway with the domain `uds2.dev` you would add the following toyour helm overrides for the `uds-dev-stack` chart:
```

Comment:
nit: if I'm not mistaken `i.e.` _typically(?)_ means "that is" or "in other words", and I think "For example" (or `E.g.`) works better here as we're giving an actual example and not simply restating the prior sentence in another way. 

```suggestion
For example, to add a custom gateway with the domain `uds2.dev` you would add the following helm overrides for the `uds-dev-stack` chart:
```

Reply:
I definitely abuse this and read as `in example` often - good callout 😅